cpp/CMakeLists.txt

```diff
@@ -62,7 +62,9 @@ if(BUILD_TYPE STREQUAL "DEBUG")
   add_compile_options(-DDEBUG )
 endif()
 
- 
+## relative path
+set(CMAKE_BUILD_RPATH "$\{ORIGIN\}")
```

Comment:
Should already be handled here, I think: https://github.com/TileDB-Inc/TileDB-CSharp/blob/main/cpp/src/CMakeLists.txt#L114

Reply:
then maybe we have to copy those two lib files to usr/lib, as suggested in https://github.com/dotnet/runtime/issues/34711